README.md

```diff
@@ -10,9 +10,14 @@ UAParser is a Julia port of [ua-parser](https://github.com/tobie/ua-parser), whi
 
 > "The crux of the original parser--the data collected by [Steve Souders](http://stevesouders.com/) over the years--has been extracted into a separate [YAML file](https://github.com/tobie/ua-parser/blob/master/regexes.yaml) so as to be reusable _as is_ by implementations in other programming languages."
 
-UAParser is a limited Julia implementation heavily influenced by the [Python code](https://github.com/tobie/ua-parser/tree/master/py) from the ua-parser library.
+UAParser is a limited Julia implementation heavily influenced by the [Python code](https://github.com/tobie/ua-parser/tree/master/py) from the ua-parser
+library.  Ideally the code would be rewritten to be more performant and Julian, but right now this is not a priority.  This should already be much more
```

Comment:
Not sure it's necessary to comment on the perceived flaws of the Python library, nor that an "ideal" is that the code be re-written.